NEWS

```diff
@@ -2,6 +2,26 @@ systemd System and Service Manager
 
 CHANGES WITH 240 in spe:
 
+        * NoNewPrivileges=yes has been set for all long-running services
```

Comment:
logind is missing right now, that part got reverted...

Reply:
NNP=yes is still set, it was "fs namespacing options" that got reverted in 28f38a76345b7548700d2337dd8b9a8c3f5b0643 ?